src/onegov/event/collections/occurrences.py

```diff
@@ -116,6 +117,10 @@ def range_to_dates(self, range, start=None, end=None):
         if range == 'month':
             start = today.replace(day=1)
             return start, start + relativedelta(months=1, days=-1)
+        if range == 'past':
+            milenium = date(2000, 1, 1)
```

Comment:
"millenium" should have two L's, but it won't appear anywhere, so it doesn't really matter.